src/components/html/EcInstalledAppCard.html

```diff
@@ -1,49 +1,109 @@
-<a-card
-  class="ec-installed-app-card"
+<div 
+  class="card border" 
```

Comment:
É bom manter a classe do nosso componente `.ec-installed-app-card`:

```suggestion
  class="ec-installed-app-card card border" 
```